site/gatsby-site/src/components/discover/hitTypes/Details.js

```diff
@@ -19,41 +20,29 @@ const StyledLabel = styled.p`
   margin: 0.6em 0;
 `;
 
-const TitleHighlight = styled(Highlight)`
-  font-weight: bold;
-`;
-
 export default function Details({
   item,
   authorsModal,
   submittersModal,
   flagReportModal,
   toggleFilterByIncidentId,
 }) {
+  console.log(item);
```

Comment:
Leftover logging?

Reply:
Good catch. I wonder if it would be worth it to add `no-console` to the eslint config for src/components and src/pages?